.github/workflows/test_weaver-fabric-fabric-data-sharing.yaml

```diff
@@ -0,0 +1,1044 @@
+# Copyright IBM Corp. All Rights Reserved.
```

Comment:
Looks like this is a redundant file, can delete this.

Reply:
It seems that this comment is obsolete now as the file has been already deleted.